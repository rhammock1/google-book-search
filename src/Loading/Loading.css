.loading-container {
  width:auto;
  height:auto;
  display:flex;
  justify-content:center;
}
#loading {
  width:100px;
}
