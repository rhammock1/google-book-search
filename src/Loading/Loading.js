import React from 'react';
import './Loading.css'

const Loading = function() {
  return (
    <div className='loading-container'>
      <img id='loading' src='https://media.giphy.com/media/QsZxCOejpGL9U48kyo/giphy.gif' alt='Loading screen' />
    </div>
  )
}

export default Loading;