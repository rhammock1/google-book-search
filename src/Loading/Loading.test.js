import React, { createElement } from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Loading from './Loading';



describe('Loading component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Loading />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the UI as expected', () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})