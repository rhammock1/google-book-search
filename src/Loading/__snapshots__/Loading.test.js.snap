// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders the UI as expected 1`] = `
<div
  className="loading-container"
>
  <img
    alt="Loading screen"
    id="loading"
    src="https://media.giphy.com/media/QsZxCOejpGL9U48kyo/giphy.gif"
  />
</div>
`;
